hadoop-ozone/pom.xml

```
@@ -26,7 +26,7 @@
 
   <properties>
     <docker.image>apache/ozone:${project.version}</docker.image>
-    <spring.version>5.2.18.RELEASE</spring.version>
+    <spring.version>5.2.20</spring.version>
```

Build fails for me due to artifact not found.  I think this should be:

```suggestion
    <spring.version>5.2.20.RELEASE</spring.version>
```